### DISCUSSION TITLE
Suggest to support Pinyin for Quick Search/Filter

### DISCUSSION THREAD

**Discussion by warkcod** [doublecmd/doublecmd#1576] [Suggestions]
When I use TC, it support quick search/Filter according to Pinyin in Chinese, such as I type 'zyhc', it will filter like fig.1 
<img width="759" alt="image" src="https://github.com/doublecmd/doublecmd/assets/131120720/1dcc3bdb-b371-486c-82d9-9c17e5f718c4">
as "怎样画出*" the first character of each word is "zyhc"

but when I use DC, when I typed 'zyhc', it filtered nothing.
<img width="779" alt="image" src="https://github.com/doublecmd/doublecmd/assets/131120720/ac9b296a-a1f3-41f7-92f7-162e2062e034">

![image](https://github.com/doublecmd/doublecmd/assets/131120720/abb8ad13-59e3-4d65-b4d1-dce9621a1f53)

**Comment by rich2014** [comment]
it's supported in DC.

which version are you using?
